lang/cs/Org.Apache.REEF.Evaluator.Tests/ContextRuntimeTests.cs

```diff
@@ -184,7 +186,17 @@ public void TestUnableToSpawnChildWhileTaskIsRunning()
 
                     Assert.True(contextRuntime.TaskRuntime.IsPresent());
                     Assert.True(contextRuntime.GetTaskStatus().IsPresent());
-                    Assert.Equal(contextRuntime.GetTaskStatus().Value.state, State.RUNNING);
+
+                    // wait for the task to start
```

Comment:
Could you use bind volatile to bind a `CountDownEvent` instead of using a timeout?